docs/source/development/adding_to_the_api_documentation.ipynb

```diff
@@ -71,7 +70,10 @@
     "    callback: Optional[Callable] = None,\n",
     "    **kwargs\n",
     ") -> Tuple[\n",
-    "    \"probnum.RandomVariable\", \"probnum.RandomVariable\", \"probnum.RandomVariable\", Dict\n",
+    "    \"randvars.RandomVariable\",\n",
+    "    \"randvars.RandomVariable\",\n",
+    "    \"randvars.RandomVariable\",\n",
+    "    Dict,\n",
```

Comment:
wow, readable notebook diff! Did you do anything special that I should adapt?

Reply:
Nope, just corrected `probnum.RandomVariable` to `randvars.RandomVariable`. Readable diffs are a consequence of not having rerun any cells that actually produce output.